features/events/api/pom.xml

```diff
@@ -83,6 +83,10 @@
       <artifactId>hibernate-validator-annotation-processor</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.opennms.core.ipc.sink</groupId>
+      <artifactId>org.opennms.core.ipc.sink.xml</artifactId>
```

Comment:
We should change this to depend on the `org.opennms.core.ipc.sink.api` instead, and add a dependency on `org.opennms.core.ipc.sink.xml` in the `features/events/daemon/pom.xml` where the module is implemented.